decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -128,30 +171,30 @@
   # that an organization's administrator injects malicious scripts to spy on or
   # take over user accounts.
   #
-  config.enable_html_header_snippets = false
+  config.enable_html_header_snippets = Rails.application.secrets.decidim[:enable_html_header_snippets].present?
```

Comment:
I think this needs another check (with `true` boolean and a string with the value `"true"` or something like that)

```ruby
> "false".present?
=> true
```

Reply:
I dont' get this one, in this case "present?" is true if any value is assigned and false if empty string or nil

Reply:
I can imagine a scenario where an implementer is seeing all the environment variables, see this one (`DECIDIM_ENABLE_HTML_HEADER_SNIPPETS`) decides that they don't want it enabled, set it to "false" and - weirdly - it's enabled even though they set it up as false. 

Maybe we should clarify it in the documentation? For what I understand, reading as it is now: 

> |Set to `true` in order to allow administrators to define an arbitrary custom HTML code in the `<head></head>` section any Decidim page.

Then if set it as `false` it should be disabled, you know what I mean? 

Doing a quick search, I found [this](https://github.com/Platoniq/decidim-install/issues/53): 

![image](https://user-images.githubusercontent.com/717367/151768311-41eb4a68-9ffb-45c6-ab8f-2350c679945d.png)

So when/if they migrate to this new configuration system, they could be bitten by this

Reply:
If we do that, I think we should detected the word "false" in all cases no? otherwise would be inconsistent. What you think?

Reply:
I prefer to check for true/"true" -- so I was thinking something like 

```suggestion
  config.enable_html_header_snippets = [true, "true"].include? Rails.application.secrets.decidim[:enable_html_header_snippets]
```

Reply:
Ok, I've gone in an alternative approach. Due the increasing complexity of processing ENV vars I've decided to create a library to parse them instead of do that inside secrets as it will make the whole file quite unreadable.

Regarding this specific case, now any value associated with "false" is treated as such:
https://github.com/decidim/decidim/blob/48b4773a3055cb8217a78de074d795e602b082e9/decidim-core/lib/decidim/env.rb#L7

This will work with ANY other ENV var, so now undefined, empty or "falsey" values have the same meaning. I think this is the best approach and the more consistent one. Not sure we really need to be that picky with this particular ENV to require exactly the "true" word (and not in the others). I think also it follows to the "principle of the least surprise" which I'm particularly very fond of.